packages/armkit-cli/test/schema.test.ts

```diff
@@ -4,42 +4,119 @@ import * as path from 'path';
 import { ImportArmSchema } from '../lib/import';
 import { Language } from '../lib/base'
 
-test('container_instance_schema.test', async () => {
-  const workdir = fs.mkdtempSync(path.join(os.tmpdir(), 'container_instance.test'));
-  const importer = new ImportArmSchema(`["2019-12-01/Microsoft.ContainerInstance"]`);
+// Comprehensive list of services.
```

Comment:
Does it makes sense to have a "Services" fixture rather than inline it here?

Reply:
refactored into a service